tutorials/quickstart-node.md

```diff
@@ -0,0 +1,497 @@
+
+Above our `app.use` statement, add the following to `index.js`:
+
+```javascript
+let PageLoads = sequelize.define('page_loads', {
+    userAgent: Sequelize.STRING,
+    time: Sequelize.DATE
+});
+```
+
+We will now be able to instantiate a `PageLoads` object and save it to the 
+database.
+
+### Generate hypertable [](create_hypertable)
+
+In TimescaleDB, the primary point of interaction with your data is a [hypertable][hypertable], 
+the abstraction of a single continuous table across all space and time 
+intervals, such that one can query it via standard SQL.
+
+Virtually all user interactions with TimescaleDB are with hypertables. Creating tables 
+and indexes, altering tables, inserting data, selecting data, etc. can (and should) 
+all be executed on the hypertable.
+
+A hypertable is defined by a standard schema with column names and types, with at 
+least one column specifying a time value.
+
+>:TIP: The TimescaleDB documentation on [schema management and indexing[indexing-api-guide] explains this in further detail. 
+
+Let's create this migration to modify the `page_loads` table and create a
+hypertable by first running the following command:
+
+```bash
+npx sequelize migration:generate --name add_hypertable
+```
+
+You should get a result that looks like this:
+
+```bash
+Sequelize CLI [Node: 12.16.2, CLI: 5.5.1, ORM: 5.21.11]
+
+migrations folder at [some_path] already exists.
+New migration was created at [some_path]/]20200601202912-add_hypertable.js .
+```
+
+And there should now be a file in your `migrations` folder that we can modify
+to look like the following:
+
+```js
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.query("SELECT create_hypertable('page_loads', 'time');");
+  },
+
+  down: (queryInterface, Sequelize) => {
+  }
+};
+```
+
+Now run the migration command from the command-line:
+
+```bash
+npx sequelize db:migrate
+```
+
+You should get the following result:
+
+```bash
+Sequelize CLI [Node: 12.16.2, CLI: 5.5.1, ORM: 5.21.11]
+
+Loaded configuration file "config/config.json".
+Using environment "development".
+== 20200601202912-add_hypertable: migrating =======
+== 20200601202912-add_hypertable: migrated (0.426s)
+```
+
+### Insert rows into TimescaleDB [](insert_rows)
+
+Now you have a working connection to your database, a table configured with 
+the proper schema, and a hypertable created to more efficiently query data 
+by time. Let's add data to the table.
+
+In the `index.js` file, we will modify the `/` route like so to first get the 
+`user-agent` from the request object (`req`) and the current timestamp. Then,
+we will save call the `create` method on our model (`PageLoads`), supplying
+the user agent and timestamp parameters:
+
+```javascript
+app.get('/', async (req, res) => {
+    // get the user agent and current time
+    const userAgent = req.get('user-agent');
+    var date = new Date();
+    var time = date.getTime();
```

Comment:
Shouldn't use `var`, use `const` instead
Also can be simplified to `const time = new Date().getTime();` pretty common to write it this way